app/src/main/java/com/azuresamples/msalnativeauthandroidkotlinsampleapp/WebFallbackFragment.kt

```diff
@@ -114,7 +114,9 @@ class WebFallbackFragment : Fragment() {
                     )
                 )
             } else {
-                    displayDialog(getString(R.string.unexpected_sdk_result_title), actionResult.toString())
+                // Please refer to other sign in fragments for the handling of other possible results.
```

Comment:
Same as others "else" block:
```suggestion
                // Unexpected result
```